diary-alexander-jaffray/modernized/15-notes-diary.adoc

```diff
@@ -224,14 +222,14 @@ is written promiscuously, Fraser, Frazer, Fraiser, Frissil, Frizell.
 
 === Note J.
 
-On an examination of the Acts and Minutes of the Scottish Parliament,
+On an examination of the _Acts and Minutes of the Scottish Parliament,_
 at the date of the 3rd of the month called February, 1646,
 Alexander Jaffray is found to be one of "`a Commission
-for Proceedings against Malignants and Delinquents.`"
-According to Balfour`'s Annals, this was a Committee for "`Fines, Processes,
+for Proceedings against __Malignants and Delinquents.__`"
+According to Balfour`'s _Annals,_ this was a Committee for "`Fines, Processes,
 and Excise.`"--Although he notices thus cursorily
 his being a member of Parliament in 1646,
-and again, at a future page, in 1649; yet, by the Acts and Minutes, it would seem,
+and again, at a future page, in 1649; yet, by the _Acts and Minutes,_ it would seem,
```

Comment:
Ok, I think this list of dates probably needs something. Why don't we try to use the syllogism class for this one (even though its obviously not a syllogism).  So that it looks like this:

```
[.syllogism]
* 1644, 5th month, (July) 19th. He is nominated with others a Commissioner for Suppressing the Rebellion, which was put down by Argyle.
* 24th. On a Committee of War for the county of Aberdeen. 1646, 12th mo: 2nd. On a similar appointment.
* etc. etc.
```

Reply:
Oh weird, I left a note about that I thought...maybe it didn't post? Idk!